### ISSUE TITLE
GallerySizeException

### ISSUE THREAD

**Issue by ZajdaBear** [allegro/allegro-api#7345]
When creating product offer, I get 422 response
``{"errors":[{"code":"GallerySizeException","message":"Unprocessable Entity","details":null,"path":"images","userMessage":"The limit of 16 images has been exceeded."}]}``

I am not sending any additional images, the issue is with product itself.

````{
    "productSet": [
        {
            "product": {
                "id": "8595226707823",
                "idType": "GTIN"
            },
            "quantity": {
                "value": 1
            }
        }
    ],
    "delivery": {
        "handlingTime": "PT0S",
        "shippingRates": {
            "id": "c19198c4-29a2-43de-9d76-8cc04d1e1a08"
        }
    },
    "publication": {
        "status": "INACTIVE"
    },
    "language": "cs-CZ",
    "afterSalesServices": {
        "impliedWarranty": {
            "id": "8ea1f88c-460e-45be-bea1-cf5d49a7b621"
        },
        "returnPolicy": {
            "id": "5c9ba402-312f-4481-8269-8c57d749a9c4"
        }
    },
    "name": "BRUCE J pergola, TRANSPARENTNÍ polykarbonát",
    "sellingMode": {
        "format": "BUY_NOW",
        "price": {
            "amount": "15000.00",
            "currency": "CZK"
        }
    },
    "stock": {
        "available": 1,
        "unit": "UNIT"
    },
    "location": {
        "city": "Jaroměř",
        "countryCode": "CZ",
        "postCode": "55101"
    },
    "external": {
        "id": "roj-2890-B618"
    },
    "messageToSellerSettings": {
        "mode": "HIDDEN"
    }
}

Please advice, how should I proceed.

**Comment by MaciejFrackowiak** [comment]
Yes, there is a bug in the product description - it has 17 images, where the limit of the offer is 16. I have reported that issue to our internal department.

**Comment by ZajdaBear** [comment]
Thanks, do you need me to send you more examples, or can you identify all of problematic products yourself?

**Comment by MaciejFrackowiak** [comment]
No, it is impossible for us to identify all the products from our product base where a similar problem exists. You can report that kind of issue by [POST /sale/products/{productId}/change-proposals](https://developer.allegro.pl/documentation/#operation/productChangeProposal) or via website - https://allegro.pl/moje-allegro/sprzedaz/produkt/zglos-blad/{productId}.

**Comment by MaciejFrackowiak** [comment]
We have fixed the product with GTIN=8595226707823.

**Comment by ZajdaBear** [comment]
Thanks
